### DISCUSSION TITLE
Remote Access to EZVIZ TY1 through HUAWEI EG8145V5

### DISCUSSION THREAD

**Discussion by kmchen13** [Motion-Project/motion#1638] [Q&A]
Hi,
I installed MOTION on Debian 11 to control an EZVIZ camera. It works fine on local network. I just had to define the cam_url as rtsp://identifier:password@192.168.1.34:554/

Now I want to access through INTERNET. It seems that I have to create a VPN to achieve this. I have an INTERNET server OPENVPN. Can I use this or do I have to add another VPN router in my local network ?

Any help apreciated

**Comment by tosiara** [comment]
You can either open a tcp port 554 using port forwarding or use vpn

**Reply by kmchen13** [reply]
I tried 554 port forwarding but access is denied
If I add an OPENVPN router on my local network, behind the HUAWEI, will I access the camera EZVIZ even if it is not an OPENVPN client ?
Will a simple TCP/UDP port forwarding from the HUAWEI to the OPENVPN server router let me access to the rtsp fglow of the camera ?

**Reply by tosiara** [reply]
you should investigate why you get access denied. it could be that you will get the same access denied when using rtsp over vpn

**Comment by kmchen13** [comment]
I don't know how to investigate that as the camera doesn't provide any log access. How could access be denied over VPN when it is opened through local network ?

**Reply by tosiara** [reply]
How could you get access denied when connecting over port forwarding?

**Comment by kmchen13** [comment]
I thought the camera's app had a way to detect the external origin of the requests.
What other cause might deny the access ?

**Comment by hormigaclub** [comment]
Tunnel through ngrok.io?

**Reply by kmchen13** [reply]
I don't understand
